pallets/inactivity-tracking/src/lib.rs

```diff
@@ -0,0 +1,297 @@
+// Copyright (C) Moondance Labs Ltd.
+// This file is part of Tanssi.
+
+// Tanssi is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Tanssi is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Tanssi.  If not, see <http://www.gnu.org/licenses/>
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Small doc about what the pallet does

Reply:
Added in https://github.com/moondance-labs/tanssi/pull/922/commits/0b111609fd8b3db997d6b05f416c833732c95de8